linter.py

```diff
@@ -2,16 +2,20 @@
 
 
 class Ameba(Linter):
-    regex = (r'^.+:(?P<line>\d+):(?P<col>\d+): ((?P<error>E)|(?P<warning>W)): (?P<message>.+)$')
+    regex = (r'^.+:(?P<line>\d+):(?P<col>\d+): ((?P<error>E)|(?P<warning>.)): (?P<message>.+)$')
```

Comment:
I think I prefer to be explicit and include the new type of offense (`convention`) instead of adding a wildcard, could you update it?

Reply:
Sure thing :+1: 

My reasoning for making it open ended is that it had become annoying to have my CI tell me of problems (given that CI was running with no ameba configurations) that the sublime linter was hiding. I'd rather it be noisier in this regard than not. I also see the reasoning for wanting the check for warnings to be explicit :) Updated!

Thanks for the correction re: correctable vs convention, it's good to know the vernacular :D